sources/en/f/freewebnovel.py

```diff
@@ -21,16 +21,26 @@ def initialize(self) -> None:
                 "p": [
                     r"freewebnovel\.com",
                     r"Updates by Freewebnovel\. com",
-                    r"” Search Freewebnovel\.com\. on google”\.",
+                    r"\” Search Freewebnovel\.com\. on google\”\.",
```

Comment:
the regex is wrong here. you do not need backslash before the quote. `\”`

please fix all the similar issues with the regex strings here.

Reply:
changed, hope it's okay now.